gateway/handler.go

```diff
@@ -269,7 +275,16 @@ func RepoOperationHandler(sc *ServerContext, repoID string, handler operations.R
 		repo, err := authOp.Cataloger.GetRepository(sc.ctx, repoID)
 		if errors.Is(err, db.ErrNotFound) {
 			authOp.Log().WithField("repository", repoID).Warn("the specified repo does not exist")
```

Comment:
warning in both cases will be probably bad on the performance

Reply:
I don't think we should warn on invalid requests or missing resources